index.html

```diff
@@ -0,0 +1,172 @@
+                 allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" 
+                allowfullscreen> </iframe>
+                <div class="iframe-discription -bottom">
+                    <h3>My Favorite VS Code Extensions</h3>
+                    <div class="iframe-discription-stats flex">
+                        <p>404,290 views</p>
+                        <p>Jan 8, 2019</p>
+                        <span>
+                            <a href="#"> <img src="https://img.icons8.com/material-rounded/24/000000/thumb-up.png" alt="like"/> 14K</a>
+                            <a href="#"> <img src="https://img.icons8.com/material-sharp/24/000000/thumbs-down.png" alt="dislike"/> 219</a> 
+                            <a href="#"> <img src="https://img.icons8.com/ios-glyphs/24/000000/forward-arrow.png" alt="sahre"/> SHARE</a>
+                            <a href="#">SAVE</a>
+                            <a href="#"> <img src="https://img.icons8.com/android/24/000000/more.png" alt="others"/> </a>
+                        </span>
+                    </div>
+                </div>
+                <div class="iframe-avatar flex">
+                    <picture class="avatar-img"> <img src="https://www.w3schools.com/howto/img_avatar.png" alt="avatar"/> </picture>
+                    <div class="avatar-disc">
+                        <strong>Colt Steele</strong>
+                        <p>115K subscribers</p>
+                        <p>Click "Show More" below for a full list of every extension I cover, as well as timestamps to jump ahead.</p>
+                        <a href="#">0:00:34</a> <span> Emmet</span>
+                        <br>
+                        <a href="#">SHOW MORE</a>
+                    </div>    
+                    <button>Subscribe</button>
+                </div>
+                <div class="loading  flex">
+                    <div class="loading-icon"> </div>
+                    <span>Loading  . . . </span>
+                </div>
+            </main>
+
+            <aside class="side-bar">
+                <div>
+                    <p>Up Next</p>
+                    <p>Autoplay</p>
+                    <div class="toggle">
+                        <div class="roundToggle"> </div>
+                    </div>
+                </div>
+               <ul>
+                   <li>
+                       <div class="suggestion">
+                           <iframe src="https://www.youtube.com/embed/iiADhChRriM" 
+                             allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" 
+                            allowfullscreen> </iframe>
+                            <div>                           
+                                <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Eius reiciendis atque est quisqu</p>
+                                <p class="tag">fun</p>
```

Comment:
- Reduce the space above this element.
![Annotation 2020-06-09 083136](https://user-images.githubusercontent.com/57726348/84101639-0b1db280-aa2c-11ea-883a-67d373f54019.jpg)